src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/requirethis/Example1.java

```diff
@@ -0,0 +1,26 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="RequireThis"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.requirethis;
+
+// xdoc section -- start
+class Example1 {
+  int a,b,c;
+
+  Example1(int a) {
+    // overlapping by constructor argument
+    this.a = a; // OK, this keyword used
```

Comment:
Please remove all OK comments, even they had explanation they are too obvious and just make noise.

Please remove `// overlapping by` comments too.

Reply:
Done.